examples/upload.js

```diff
@@ -0,0 +1,59 @@
+// To run this script, do `node upload.js`
```

Comment:
This file is currently marked executable. For it to be so, I think we need a hashbang, or else we get a bunch of errors like

```
./upload.js: line 1: //: Is a directory
./upload.js: line 2: //: Is a directory
./upload.js: line 3: //: Is a directory
./upload.js: line 4: //: Is a directory
./upload.js: line 5: //: Is a directory
./upload.js: line 7: syntax error near unexpected token `('
./upload.js: line 7: `const permSdk = require('@permanentorg/node-sdk');'
```

```suggestion
#!/usr/bin/env nodejs

// To run this script, do `node upload.js`
```

Happily, nodejs seems to know to ignore the hashbang. There's an interesting rabbit hole to fall down: [a proposal to make a leading hashbang part of the language](https://github.com/tc39/proposal-hashbang).

Anyway, the alternative is to not mark this file as executable, and leave the existing comment.